pythran/config.py

```diff
@@ -136,9 +135,6 @@ def fmt_define(define):
         output.append('-L' + distutils.sysconfig.get_config_var('LIBPL'))
         output.extend(('-l' + include)
                       for include in extension['libraries'])
-        output.extend(distutils.sysconfig.get_config_var('LIBS').split())
-        output.append('-lpython' +
-                      distutils.sysconfig.get_config_var('VERSION'))
```

Comment:
You remove it here but you didn't add it in the default compilation. It won't compile no?